docs/v1/api.md

```diff
@@ -0,0 +1,230 @@
+# Service Catalog API
+
+This document proposes a Kubernetes-native service broker API. It is a formalization of [@krancour](http://github.com/krancour)'s [presentation](https://docs.google.com/presentation/d/1Fm3qG9zQ4R8ZbhtTObmQjO3UZYJ0ruYI4qr5w_gEKN8/edit?usp=sharing) that he gave at the sig-service-catalog meeting on 10/10/2016.
```

Comment:
per discussion on the #sig-service-catalog channel in the kubernetes slack, I've changed language in this document to indicate that it's a design doc, not a proposal in https://github.com/kubernetes-incubator/service-catalog/pull/13/commits/4fd96b3584cf8890ff24e57d9a2f6c602e82dbda